utilities/rpp-unittests/HOST/Single_host.cpp

```diff
@@ -0,0 +1,515 @@
+        break;
+    case 6:
+        test_case_name = "gamma_correction";
+        rppi_gamma_correction_u8_pkd3_host(input, srcSize[0], output, gamma, handle);
+        break;
+    case 7:
+        test_case_name = "fog";
+        rppi_fog_u8_pkd3_host(input, srcSize[0], output, fogValue, handle);
+        break;
+    case 8:
+        test_case_name = "snow";
+        rppi_snow_u8_pkd3_host(input, srcSize[0], output, snowPercentage, handle);
+        break;
+    case 9:
+        test_case_name = "lens_correction";
+        rppi_lens_correction_u8_pkd3_host(input, srcSize[0], output, strength, zoom, handle);
+        break;
+    case 10:
+        test_case_name = "noise";
+        rppi_noise_u8_pkd3_host(input, srcSize[0], output, noiseProbability, handle);
+        break;
+    case 11:
+        test_case_name = "pixelate";
+        rppi_pixelate_u8_pkd3_host(input, srcSize[0], output, handle);
+        break;
+    case 12:
+        test_case_name = "exposure";
+        rppi_exposure_u8_pkd3_host(input, srcSize[0], output, exposureFactor, handle);
+        break;
+    case 13:
+        test_case_name = "fisheye";
+        rppi_fisheye_u8_pkd3_host(input, srcSize[0], output, handle);
+        break;
+    case 14:
+        test_case_name = "vignette";
+        rppi_vignette_u8_pkd3_host(input, srcSize[0], output, stdDev, handle);
+        break;
+    case 15:
+        test_case_name = "flip";
+        rppi_flip_u8_pkd3_host(input, srcSize[0], output, flipAxis, handle);
+        break;
+    case 16:
+        test_case_name = "rain";
+        rppi_rain_u8_pkd3_host(input, srcSize[0], output, rainPercentage, rainWidth, rainHeight, transparency, handle);
+        break;
+    case 17:
+        test_case_name = "rotate";
+        rppi_rotate_u8_pkd3_host(input, srcSize[0], output, dstSize[0], angle, handle);
+        break;
+    case 18:
+        test_case_name = "warp-affine";
+        rppi_warp_affine_u8_pkd3_host(input, srcSize[0], output, dstSize[0], affine, handle);
+        break;
+    case 19:
+        test_case_name = "resize";
+        rppi_resize_u8_pkd3_host(input, srcSize[0], output, dstSize[0], handle);
+    case 20:
+        test_case_name = "resize_crop";
+        rppi_resize_crop_u8_pkd3_host(input, srcSize[0], output, dstSize[0], coordinates[0], coordinates[1],
+                                      coordinates[2], coordinates[3], handle);
+    case 21:
+        test_case_name = "Hue modification";
+        rppi_hueRGB_u8_pkd3_host(input, srcSize[0], output, hueShift, handle);
+        break;
+    case 22:
+        test_case_name = "Saturation";
+        rppi_saturationRGB_u8_pkd3_host(input, srcSize[0], output, saturationFactor, handle);
+        break;
+    case 23:
+        test_case_name = "Histogram Balance";
+        rppi_histogram_balance_u8_pkd3_host(input, srcSize[0], output, handle);
+        break;
+    case 24:
+        test_case_name = "RandomShadow";
```

Comment:
Codacy found an issue: [Variable 'test_case_name' is assigned a value that is never used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)